tests/update/patch_sle.pm

```diff
@@ -96,6 +96,11 @@ sub patching_sle {
 
     assert_script_run("zypper mr --enable --all");
 
+    #removing old repositories during AutoYaST driven upgrade
```

Comment:
This is 'disable' rather than 'remove', right? so please change this comment as well as PR title and commit message.
Also fix the comment style where you can follow others in this pm.